core/renderers/measurables/row_elements.js

```diff
@@ -125,7 +125,7 @@ Blockly.blockRendering.RoundCorner = function(opt_position) {
   this.width = this.constants_.CORNER_RADIUS;
   // The rounded corner extends into the next row by 4 so we only take the
   // height that is aligned with this row.
-  this.height = this.constants_.NOTCH.height;
+  this.height = this.constants_.CORNER_RADIUS / 2;
```

Comment:
It would be nice if we could get to a point where the corners store their true height value. 
The corner extending past its row, that statement isn't true for zelos. What's our plan here? Is there value in having this report its real height if the notch height is greater?

Reply:
Yeah, that gets back to whether anyone other than a notch and hat get ascenders and descenders.

In either case, the notch height should have nothing to do with the corner height.  That was just a convenient way of getting a 4 in the past.